Sources/CoreCommands/SwiftTool.swift

```diff
@@ -275,6 +275,7 @@ public final class SwiftTool {
     internal init(
         outputStream: OutputByteStream,
         options: GlobalOptions,
+        shouldInstallSignalHandlers: Bool = true,
```

Comment:
we can consider adding this to global options

Reply:
`GlobalOptions` is dedicated to properties using `@Option` property wrappers passed from ArgumentParser. Would this `ToolWorkspaceConfiguration` type values of which are passed as arguments to this initializer also work?